activation.php

```diff
@@ -12,6 +12,40 @@ function graphql_activation_callback() {
 		return;
 	}
 
+	// Run any activation scripts before updating the version.
+	graphql_migrate_1_20_0();
```

Comment:
Does this run for folks that update via composer too? 🤔

Reply:
Now that you mention it it might not even work on plugin _update_ and only on actual activation. 🤦

Not sure the best approach here:
- If it's non-composer, we could conditional it as a run_once on `admin_init`, but someone updating via composer would still need to log in. 
- Or we absorb the db version check on every `init`, the extra potential hit is annoying, but still less than with QA activated.
- Or we just keep the setting and filter logic, and wait until 2.0 to switch the the default behavior to `off`.


@jasonbahl thoughts?

Reply:
@justlevine perhaps in the `is_enabled_for_query` we can add some logic to handle folks that might not have triggered the `migrate` script. 

Like, when we determine if GRAPHQL_DEBUG is enabled, etc. . .

If we are able to determine no value has been set in the db for `query_analyzer_enabled` we can assume the migrate script hasn't run and/or the user hasn't manually saved the settings yet, so we can have some logic there to compensate for users that didn't trigger the migrate script?